ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -105,15 +106,18 @@ private final case class ParsedPackageData(
 private final case class ParsedCommandData(deduplicateUntil: Instant)
 
 private class JdbcLedgerDao(
+    override val maxConnections: Int,
     dbDispatcher: DbDispatcher,
     contractSerializer: ContractSerializer,
     transactionSerializer: TransactionSerializer,
     keyHasher: KeyHasher,
     dbType: DbType,
-    executionContext: ExecutionContext,
+    executor: Executor,
 )(implicit logCtx: LoggingContext)
     extends LedgerDao {
 
+  private implicit val ec: ExecutionContext = ExecutionContext.fromExecutor(executor)
```

Comment:
I prefer the previous approach of keeping it explicit. It's not too elegant but it clarifies where we switch execution context.

Reply:
I agree with @stefanobaghino-da. Let's keep it explicit as much as possible.

Reply:
Threw away the commit that makes things implicit.